src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from "react";
+import "./navbar.scss";
+
+class Navbar extends Component {
+    render(){
+        let home="/", blog="/", bibliography = "/";
+        let contact="/", join="/";
+
+        return (
+            <nav className="navbar navbar-fixed">
+                <a href={home} className="nav-item">
```

Comment:
Acredito que deveríamos utilizar um componente `Link` para isto, como [indicado na documentação](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/);